feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -87,20 +88,33 @@ func TestOSInstall(t *testing.T) {
 	}
 	tc.fetchStandbySupervisorStatus(ctx, t)
 	tc.transferOS(ctx, t, false)
-	tc.activateOS(ctx, t, false)
-	if tc.dualSup {
+
+	if *deviations.OSActiavteRequiresReboot {
```

Comment:
This deviation only applies to duaSup case, right? Which means you probably need the if `if tc.dualSup` condition?

Reply:
no, it applies for both cases since some vendors do not support activating os without doing reboot.